ppl/src/main/antlr/OpenSearchPPLLexer.g4

```diff
@@ -363,12 +363,14 @@ Y:                                  'Y';
 // LITERALS AND VALUES
 //STRING_LITERAL:                     DQUOTA_STRING | SQUOTA_STRING | BQUOTA_STRING;
 ID:                                 ID_LITERAL;
+CLUSTER:                            CLUSTER_PREFIX_LITERAL;
```

Comment:
can we unify the external source `DATASOURCES` with the `CLUSTER` into one logical concept ?

Reply:
multi-DS  and multi-cluster queries are very different technically. Different engines have different capabilities